.gitattributes

```diff
@@ -11,3 +11,5 @@
 *.y text
 demo/primer.css binary
 demo/libwabt.js binary
+demo/third_party/codemirror/codemirror.css binary
+demo/third_party/codemirror/codemirror.js binary
```

Comment:
I'm not sure that not wanting to see the diffs is a good reason for marking something as binary.   Binary should be for binaries.